integration-templates/bamboohr-basic/nango.yaml

```diff
@@ -4,19 +4,17 @@ integrations:
       bamboohr-employees:
         runs: every 6 hours
         description: |
-          Fetches employees from bamboohr.
-          Details: full sync, doesn't track deletes, metadata is not required.
```

Comment:
I personally don't find it useful but I'm not sure it's up to us to decide, is this coming from a Linear task?

Reply:
Still good to have a short description rather than nothing at all. I think this is fine

Reply:
Ah right, that isn’t needed anymore especially since we have dedicated fields for that stuff now